From 775a6ac7e5ec5979f8631aa18d279b73561ac34f Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <yasufum.o@gmail.com>
Date: Mon, 3 Jun 2019 13:11:56 +0900
Subject: [PATCH 1/1] maintainers: update email address

For changing email usage policy in NTT, Yasufumi's address
`ogawa.yasufumi@lab.ntt.co.jp` was discarded. Because using `git
send-email` is not allowed for new account, change the email address
to `yasufum.o@gmail.com` instead of company's email.

Signed-off-by: Yasufumi Ogawa <yasufum.o@gmail.com>
---
 MAINTAINERS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index 2f7128b..b848ae6 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -24,5 +24,5 @@ General Project Administration
 
 Main Branch
 M: Ferruh Yigit <ferruh.yigit@intel.com>
-M: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
+M: Yasufumi Ogawa <yasufum.o@gmail.com>
 T: git://dpdk.org/apps/spp
-- 
2.17.1

